### DISCUSSION TITLE
Setup universe how to ?

### DISCUSSION THREAD

**Discussion by ghdc-ispm** [r-universe-org/help#472] [Q&A] [answered]
At this link to the documentation : https://github.com/r-universe-org/help

It is said to create a repo named '<username>.r-universe.dev' but in some other sources for example this one https://ropensci.org/commcalls/may2021-r-universe/, the repo should be named simply 'universe' so I'm confused, which one is correct ?

Also I suppose this repo must be a public one ?

**Comment by jeroen** [comment, marked as answer]
Yes it was changed last year, see: https://ropensci.org/blog/2023/02/07/runiverse-registry-repo/
It has to be `{ghname}.r-universe.dev`. And yes it must be public.

**Reply by jeroen** [reply]
You also still need to install the github app to activate your universe: https://github.com/apps/r-universe/installations/new
